Kernel/API/InodeWatcherEvent.h

```diff
@@ -1,21 +1,32 @@
 /*
- * Copyright (c) 2020, the SerenityOS developers.
+ * Copyright (c) 2020-2021, the SerenityOS developers.
  *
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
 #pragma once
 
+#include <AK/EnumBits.h>
 #include <AK/Types.h>
+#include <limits.h>
```

Comment:
It does work, I didn't have any issues with it. I think this includes from `LibC` during the kernel build and userland, and the system header during Lagom build.
`NAME_MAX` is required to calculate the maximum size of an event. I could hardcode it here too, but I'd like to avoid that if possible. :^)

Reply:
Yeah it's definitely better to include it for `NAME_MAX`, I'm just.. surprised.. that it works.